recipes/slave.rb

```diff
@@ -80,7 +80,7 @@ class Chef::Recipe
     exec_start '/etc/mesos-chef/mesos-slave'
     restart 'on-failure'
     restart_sec 20
-    limit_nofile 16384
+    limit_nofile 65536
```

Comment:
Instead of setting hard value in cookbbok, we should make this configurable so that we don't have to patch the upstream cookbook each time we want to update the value.

Reply:
systemd has its own way to override parameters when needed using dropin files (which would not require to change this cookbook). I don't think there is a need to add another attribute that will make the interface more complex overall.